stackstate_checks_dev/stackstate_checks/dev/docker.py

```diff
@@ -213,7 +213,7 @@ def __init__(self, compose_file, build=False, service_name=None):
             self.command.append(self.service_name)
 
     def __call__(self):
-        return run_command(self.command, check=True)
+        return run_command(self.command, "both", check=True)
```

Comment:
what does `both` do here?

Reply:
it makes sure we get stdout and stderr from the docker compose